### PR TITLE
Feat/accessibility

### PR DESCRIPTION
**Pull request by benmb1984**

### CODE REVIEW

File: portal/App.js [hunk 1]
```diff
@@ -77,7 +77,8 @@ import BusinessMarketMd from './../packages/theme/business-market/BUSINESS-MARKE
 
 // App context
 const AppContext = React.createContext({
-  sideBarActive: false
+  blackAndWhiteActive: false,
+  sideBarActive: false,
```
Comment (coenkoopmans) [inline portal/App.js]: Shouldn't we make this an enum? ColorMode.Default, ColorMode.Greyscale, ColorMode.Protanopia, etc?

**Review by coenkoopmans — APPROVED** [review]
It's fine for now, but pls think about the enum thing

**Review by jordybulten — APPROVED** [review]

File: portal/App.js (hunk 1 above)
Comment (jordybulten) [reply portal/App.js]: Only needed if we have multiple eye-disorders, i think...

Comment (benbesuijen) [reply portal/App.js]: Yes, when we add more modes that will be great: https://reactjs.org/docs/typechecking-with-proptypes.html

**Pull request merged**